compose.yml

```diff
@@ -71,6 +71,7 @@ services:
       - DATABASE_URL=postgresql://orcabus:orcabus@db:5432/filemanager
       - RUST_LOG=debug
       - FILEMANAGER_API_CORS_ALLOW_ORIGINS=${FILEMANAGER_API_CORS_ALLOW_ORIGINS:-http://localhost:8400}
```

Comment:
Perhaps this could be `http://localhost:3000`, as we set the local port for UI to 3000.